src/dotnet/APIView/APIViewWeb/Controllers/CopilotCommentsController.cs

```diff
@@ -0,0 +1,86 @@
+using System.Text.Json;
+using System.Threading.Tasks;
+using APIViewWeb.Managers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace APIViewWeb.Controllers
+{
+    public class CopilotCommentsController : Controller
+    {
+        private readonly ICopilotCommentsManager _copilotManager;
+        private readonly ILogger _logger;
+
+        public CopilotCommentsController(ICopilotCommentsManager copilotManager, ILogger<CopilotCommentsController> logger)
+        {
+            _copilotManager = copilotManager;
+            _logger = logger;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult> CreateDocument(string badCode, string language, string goodCode = null, string comment = null, string[] guidelineIds = null)
+        {
+            if (badCode != null)
```

Comment:
Nit, but could we put the failure case at the top?  Same goes for all other methods.